### PR TITLE
adding 5.4.3 release notes, removed 4.5.11 from the about releases pa…

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**
…ge because it is EOS. Also changed references from  128T to SSR in several other docs.

### CODE REVIEW

File: docs/release_notes_128t_5.4.md [hunk 1]
```diff
@@ -24,6 +24,50 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+
+### Resolved Issues
+
+- **I95-33730 The `Completed in .* seconds` includes the time waiting for the user to confirm:** The execution time now only includes the time spent executing a command.  
+------
+- **I95-41108 EthoSVR doesn't restart the session when all peer paths are disabled:** This is not specific to EthoSVR and can happen to any session that is unidirectional. This issue has been resolved. 
+------
+- **I95-41320 When quitting the PCLI, a slow PCLI response may result in traceback messages:** This is not harmful, and this situation is now handled internally. 
+------
+- **I95-42364 `off-subnet-reverse-arp-mac-learning` does not work when `outbound-only` is configured:** The  learned MAC is now saved in the reverse flow placeholder for use when the flow is established.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.4.md:43]: extra space after "The  "

**Review by MichaelBaj — APPROVED** [review]

File: docs/release_notes_128t_5.4.md [hunk 2]
```diff
@@ -24,6 +24,50 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
 ------
 - **Plugin Upgrades:** If you are running with plugins, updates are required for some plugins **before** upgrading the conductor to SSR version 5.4.0. Please review the [Plugin Configuration Generation Changes](intro_upgrade_considerations.md#plugin-configuration-generation-changes) for additional information.  
 
+## Release 5.4.3-1
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.4.md:27]: Build number of `5.4.3-3`

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

File: docs/release_notes_128t_5.4.md [hunk 3]
```diff
@@ -26,7 +26,7 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
 
 ## Release 5.4.3-4
```
Comment (Bhautik) [inline docs/release_notes_128t_5.4.md:27]: Release will be: 5.4.3-8

File: docs/about_releases.mdx [hunk 4]
```diff
@@ -24,11 +24,12 @@ Starting with version `5.4.0` the `-build` will reflect the number of iterations
 | --- | --- |
 | Release 5.1 | [5.1.8](release_notes_128t_5.1.md#release-518) |
 
+
 ## Limited Availability 
 
 | Version | Latest LA Version |
 | --- | --- |
-| Release 5.4 | [5.4.2](release_notes_128t_5.4.md#release-542-5) |
+| Release 5.4 | [5.4.3](release_notes_128t_5.4.md#release-543-3) |
```
Comment (Bhautik) [inline docs/about_releases.mdx:32]: Release: 5.4.3-8

**Review by Bhautik — CHANGES_REQUESTED** [review]

**Review by Bhautik — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

File: docs/release_notes_128t_5.4.md [hunk 5]
```diff
@@ -24,6 +24,50 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
 ------
 - **Plugin Upgrades:** If you are running with plugins, updates are required for some plugins **before** upgrading the conductor to SSR version 5.4.0. Please review the [Plugin Configuration Generation Changes](intro_upgrade_considerations.md#plugin-configuration-generation-changes) for additional information.  
 
+## Release 5.4.3-3
```
[the remark this answers is not in the document]
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.4.md]: You are looking at an old commit. That was changed in the checkin 2c9c5db.

**Review by jeffmart — APPROVED** [review]

**Pull request merged**